src/non-matching-media-query-remover.js

```diff
@@ -7,7 +7,8 @@ const debuglog = debug('penthouse:preformatting:nonMatchingMediaQueryRemover')
 // only filters out:
 //  - @print
 //  - min-width > width OR min-height > height
-// and the latter only if !keepLargerMediaQueries (which is the default)
+//    (the latter only if !keepLargerMediaQueries -- which is the default)
+//  - min-width > width AND max-width < width
```

Comment:
should this comment read ...`AND max-width > width`?

Reply:
yes, definitively a typo